src/uucore/src/lib/features/entries.rs

```diff
@@ -268,3 +303,23 @@ pub fn usr2uid(name: &str) -> IOResult<uid_t> {
 pub fn grp2gid(name: &str) -> IOResult<gid_t> {
     Group::locate(name).map(|p| p.gid())
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn do_vecs_match<T: PartialEq>(a: &Vec<T>, b: &Vec<T>) -> bool {
+        let matching = a.iter().zip(b.iter()).filter(|&(a, b)| a == b).count();
+        matching == a.len() && matching == b.len()
```

Comment:
Why is this necessary? `Vec` implements `PartialEq`, right?

Reply:
I tested locally and
```rust
assert_eq!(get_groups_gnu(Some(last)).unwrap(), groups);
```
should work

Reply:
thanks!